<?xml version="1.0"?>
<!-- Test launch for the sample_detection node -->
<!-- vim: set fenc=utf-8 et sts=4 ft=xml : -->
<launch>
    <machine name="navigate"
             address="localhost"
             env-loader="/home/zlizer/src/SampleReturn/ros_workspace/devel/env.sh"
             user="zlizer"
    />
  <include file="$(find samplereturn)/launch/navigation_camera.launch">
    <arg name="run_cameras" value="false"/>
  </include>

    <node
      pkg="sample_detection"
      type="color_name_sample_detection.py"
      name="color_name_sample_detection"
      ns="sample_detection">
      <param name="color_file" type="string" value="$(find sample_detection)/config/w2c.mat"/>
      <param name="delta" type="double" value="20.0"/>
      <param name="max_area" type="double" value="10000.0"/>
      <param name="min_area" type="double" value="200.0"/>
      <remap from="left_img" to="/navigation/left/image_rect_color"/>
      <!--
      <remap from="disparity" to="/navigation/disparity"/>
      <remap from="cam_info" to="/navigation/left/camera_info"/>
      -->
    </node>
    <node
      pkg="image_view"
      type="image_view"
      name="image_view">
      <remap from="image" to="/navigation/left/image_rect_color"/>
    </node>
    <node
      pkg="image_view"
      type="image_view"
      name="debug_image_view">
      <remap from="image" to="/sample_detection/debug_img"/>
    </node>
</launch>
